### PR TITLE
gh-65 Add prettier eslint plugin to webapp-next

### PR DESCRIPTION
**Pull request by Murad013**
🚀 Description

Added Eslint Plugin Prettier to webapp-next directory

### CODE REVIEW

**Comment by ronakmehtav** [comment]
does it not give any linting issue ?

**Comment by ronakmehtav** [comment]
> you haven't added the prettier to eslint rc. please modify it.

and eslint-prettier plugin should be dev dependency should it not ?

**Comment by ronakmehtav** [comment]
> 🚀 Description
> 
> Added Eslint Plugin Prettier to webapp-next directory
- please modify the eslint config appropriately.
- please add the eslint plugin as dev dependency.

**Comment by Murad013** [comment]
Hey @ronakmehtav I'm pretty sure I've made the proper changes. Please check them out at your earliest convenience and thank you for your patience!

**Comment by ronakmehtav** [comment]
> Hey @ronakmehtav I'm pretty sure I've made the proper changes. Please check them out at your earliest convenience and thank you for your patience!

the eslint are already under devdependency only so add the prettier plugin dev dependency.

**Comment by ronakmehtav** [comment]
> Hey @ronakmehtav I'm pretty sure I've made the proper changes. Please check them out at your earliest convenience and thank you for your patience!

and have you run next build to catch any error's 
link : https://nextjs.org/docs/basic-features/eslint

**Comment by ronakmehtav** [comment]
> Hey @ronakmehtav I'm pretty sure I've made the proper changes. Please check them out at your earliest convenience and thank you for your patience!

you forgot to add prettier plugin

**Comment by ronakmehtav** [comment]
> Hey @ronakmehtav I'm pretty sure I've made the proper changes. Please check them out at your earliest convenience and thank you for your patience!

@Murad013  also you 'll have to create prettier config file. Ask @codicocodes  about the rules he want's on the prettier config.

File: packages/webapp-next/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,7864 @@
+{
```
Comment (codicocodes) [inline packages/webapp-next/package-lock.json:1]: Please remove `package-lock.json`, speedtyper.dev uses yarn

File: packages/webapp-next/yarn.lock [hunk 2]
```diff
@@ -3,1625 +3,1577 @@
 
 
 "@babel/runtime-corejs3@^7.10.2":
-  version "7.18.9"
```
Comment (codicocodes) [inline packages/webapp-next/yarn.lock:6]: Please run `yarn` to ensure yarn.lock is up to date, when running yarn on this branch there are a lot of diffs.

**Review by codicocodes — CHANGES_REQUESTED** [review]
Thank you so much for the help @Murad013 

There are a couple of issues before we can merge this PR, please have a look!

**Comment by codicocodes** [comment]
@Murad013 I think we should go without a prettier config, if we don't have any custom rules and if it's not necessary.

Currently build is failing because the prettier config is empty. I also expect the build to fail due to unaddressed prettier issues even after that. (Just running prettier on all erroring files should suffice)

File: packages/webapp-next/.prettierrc.json [hunk 3]
```diff
@@ -0,0 +1 @@
+��
```
Comment (codicocodes) [inline packages/webapp-next/.prettierrc.json:1]: This file needs to be removed or a valid config should be added. I think we can remove the config file and go with default config for now.

**Comment by Murad013** [comment]
I rebased on main but I'm not sure what to do next 😅

**Comment by codicocodes** [comment]
> I rebased on main but I'm not sure what to do next 😅

1. First you need to make sure your main is up to date.
2. `git rebase main` -> this should cause conflicts
3. Solve the conflicts
4. force push

**Review by codicocodes — APPROVED** [review]
Thank you for the help @Murad013

**Pull request merged**